src/com_manager.c

```diff
@@ -636,6 +644,10 @@ int processIsb(unsigned int pHandle, is_comm_instance_t *comm)
             cmInstance->disableBcastFnc(-1);
         }
         sendAck(cmInstance, pHandle, &(comm->rxPkt), PKT_TYPE_ACK);
+
+        #ifdef IMX_5
```

Comment:
This is a little concerning that there are a number of these in com_manager.c; 1-liners that are enclosed in "#ifdef IMX_5" .  It makes me wonder if there is a more appropriate place to put this code that is actually IMX specific?